compat/src/suspense.js

```diff
@@ -0,0 +1,143 @@
+import { Component, createElement, unmount } from 'preact';
+import { removeNode } from '../../src/util';
+
+export function catchRender(error, component) {
+	// thrown Promises are meant to suspend...
+	if (typeof error.then === 'function') {
```

Comment:
Just reasking, you can resolve if checked but e.then gave no size benefit?

Reply:
Will re-check but IIRC no

Reply:
It cut 4 bytes. I guess my statement was based on the state when this was part of the core. Thx!